ui/components/ui/chip/chip.scss

```diff
@@ -37,7 +37,20 @@
     }
   }
 
-
+  &__input {
```

Comment:
one more minor change req:

make this &--with-input &__input {

just so that the styles are specific to the ChipWithInput component